### PR TITLE
remove `draw_item_selection_rect`

### PR DESCRIPTION
**Pull request by 0x00b1**

### CODE REVIEW

File: cellprofiler/gui/imagesetctrl.py [hunk 1]
```diff
@@ -1019,8 +1018,6 @@ def Draw(self, grid, attr, dc, rect, row, col, is_selected):
                 grid, grid.GetGridWindow(), grid.GetGridColLabelWindow(),
                 grid.GetGridRowLabelWindow(), grid.GetGridCornerLabelWindow()):
                 flags += wx.CONTROL_FOCUSED
-            draw_item_selection_rect(
```
Comment (0x00B1) [inline cellprofiler/gui/imagesetctrl.py]: use `wx.RendererNative.DrawItemSelectionRect`.

File: cellprofiler/gui/pathlist.py [hunk 2]
```diff
@@ -642,11 +641,6 @@ def on_paint(self, event):
                         flags += wx.CONTROL_SELECTED
                     if idx == self.focus_item:
                         flags += wx.CONTROL_CURRENT
-                    draw_item_selection_rect(
```
Comment (0x00B1) [inline cellprofiler/gui/pathlist.py]: use `wx.RendererNative.DrawItemSelectionRect`

File: cellprofiler/gui/pipelinelistview.py [hunk 3]
```diff
@@ -1521,7 +1520,6 @@ def on_paint(self, event):
                     if (self.always_draw_current_as_if_selected and
                         not item.selected):
                         flags |= wx.CONTROL_SELECTED
-                draw_item_selection_rect(self, dc, r, flags)
```
Comment (0x00B1) [inline cellprofiler/gui/pipelinelistview.py]: use `wx.RendererNative.DrawItemSelectionRect`

**Comment by AnneCarpenter** [comment]
@0x00b1 Does someone need to be assigned to review this PR?

**Comment by mcquin** [comment]
@0x00b1 bump

screen shot 2017-01-06 at 4 01 51 pm

🤔

**Pull request closed without merging**